template/.stylelintrc

```diff
@@ -37,7 +37,9 @@
     "media-query-list-comma-newline-before": "never-multi-line",
     "named-grid-areas-no-invalid": true,
     "no-empty-first-line": true,
+    "no-empty-source": null,
     "no-unknown-animations": true,
+    "no-invalid-position-at-import-rule": null,
```

Comment:
Are these new lines necessary?

None of the stylelint rules we already have or import set a value for those so aren't they already null?

Reply:
These are needed. These new lines were added to the custom elements that already have this feature installed.